lib/web/mfa.go

```diff
@@ -75,6 +75,41 @@ func (h *Handler) deleteMFADeviceWithTokenHandle(w http.ResponseWriter, r *http.
 	return OK(), nil
 }
 
+type deleteMfaDeviceRequest struct {
```

Comment:
```suggestion
type deleteMFADeviceRequest struct {
```

Acronyms should be capitalized in Go code.

Reply:
👍 3b0172f378d13cf47a64beed55031add88246ab9